python/server/ServerWrapper.cpp

```diff
@@ -906,6 +945,22 @@ namespace {
         return field->ID();
     }
 
+    // Wrappers for Object class member functions
```

Comment:
That comment is misleading, `ObjectsGetSystems` isn't a member function of the `UniverseObject` class (wrappers for those start around line 550 by the way, so if you add a wrapper for one of them you should place it there), as it acts on a *list* of universe objects, so to speak.

If anything, it maybe can be counted with the wrappers for member functions of the `Universe` class, because it kind of acts on the game universe. Insofar this wrapper can be considered to be in the right place already. Just adjust the comment, or remove it.